src/main/java/com/javadeobfuscator/deobfuscator/Deobfuscator.java

```diff
@@ -571,7 +577,7 @@ public ClassTree getClassTree(String classNode) {
                 }
             });
         }
-        ClassWriter writer = new CustomClassWriter(0);
+        ClassWriter writer = new CustomClassWriter(ClassWriter.COMPUTE_FRAMES);
```

Comment:
COMPUTE_FRAMES requires all libs to be added afaik. I believe thiy was not used on purpose as this is not needed for many deobfuscation steps.